.buildkite/pipeline.yml

```diff
@@ -150,7 +151,6 @@ steps:
     depends_on: ligo-test
     commands:
       - mkdir assets
-      - buildkite-agent artifact download baseDAO.tz assets --step "build-ligo"
```

Comment:
Same here, why not add the compiled contracts from the artifacts to the auto pre-release?

Reply:
Have made a change to addres this.